server/graphql/v2/mutation/MemberMutations.js

```diff
@@ -161,6 +162,20 @@ const memberMutations = {
         throw new ValidationFailed(`Member ${memberAccount.slug} does not exist in Collective ${account.slug}`);
       }
 
+      if ([MemberRoles.ACCOUNTANT, MemberRoles.ADMIN, MemberRoles.MEMBER].includes(this.role)) {
```

Comment:
It seems that we've broken this condition in the latest commits (needs to use `args`, not `this` here). I'm pushing a fix in https://github.com/opencollective/opencollective-api/pull/7598.